Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -923,7 +923,7 @@ void CGameListCtrl::OnMouseMotion(wxMouseEvent& event)
       GetItemRect(item, Rect);
       int mx = Rect.GetWidth();
       int my = Rect.GetY();
-#ifndef __WXMSW__
+#if !defined(__WXMSW__) && !defined(__WXMAC__)
```

Comment:
Good catch! I didn't know about that. Guess they sorta had the rug pulled out from them by Apple on the naming though, huh...